From e992095cb88ee315d5688f04adf5cb505db4746d Mon Sep 17 00:00:00 2001
From: Jianing Ren <jianing.ren@rock-chips.com>
Date: Sat, 21 Dec 2019 19:50:49 +0800
Subject: [PATCH] arm64: dts: rockchip: disable regulator of usbnet for
 rk3399-evb-ind

RK3399 EVB-IND Android10 SDK does not need usbnet, so we should close the
regulator of usbnet. However, regulator-always-off is not a correct
property of dts which can lead to xhci die and PM suspend fail.

Change-Id: I8e0ceafd8ee92d78724f423aecb769591716c017
Signed-off-by: Jianing Ren <jianing.ren@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-evb-ind.dtsi | 2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-evb-ind.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-evb-ind.dtsi
index f895508d47e9..cc4643dee620 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-evb-ind.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-evb-ind.dtsi
@@ -200,8 +200,6 @@
 		pinctrl-0 = <&usbnet_pwr_drv>;
 		regulator-name = "vcc5v0_usbnet";
 		startup-delay-us = <20000>;
-		regulator-always-off;
-		regulator-boot-on;
 		regulator-min-microvolt = <5000000>;
 		regulator-max-microvolt = <5000000>;
 		status = "okay";
-- 
2.35.3

